recipes/relocate2/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+mkdir $PREFIX/scripts
+cp ./scripts/*.p* $PREFIX/scripts
+cp ./scripts/*.p* $PREFIX/bin
+cp ./README.md $PREFIX
+cp ./test_data.tar.gz $PREFIX
+echo "bwa=$PREFIX/bin/bwa" >> $PREFIX/CONFIG
+echo "bowtie2=$PREFIX/bin/bowtie2" >> $PREFIX/CONFIG
+echo "bowtie2_build=$PREFIX/bin/bowtie2-build" >> $PREFIX/CONFIG
+echo "blat=$PREFIX/bin/blat" >> $PREFIX/CONFIG
```

Comment:
I guess the problem is here $PREFIX will be a full path and this will not be the same on all systems.
Is this file really needed?